src/views/kbv/abandon.njk

```diff
@@ -1,13 +1,26 @@
 {% extends "base-form.njk" %}
+{% from "components/onPageLoad/macro.njk" import ga4OnPageLoad %}
 {% set hmpoPageKey = "abandonCheck" %}
 {% set isPageHeading = true %}
 
 {% block mainContent %}
   {{ super() }}
 
   {{ hmpoHtml(translate("pages.abandonCheck.contact")) }}
-{% endblock %}
```

Comment:
Did you mean to remove this `endblock`?

The `submitButton` block is called out separately to add an `id` onto it so it can be targetted for automation.

I think your `ga4OnPageLoad` changes  would be better placed after the question content has been rendered by the `hmpoHtml` command

Reply:
Updated to move the submit button back below the end of main content. The GA4 script itself won't work outside of a nunjucks block. The current configuration works but if required, we can move it into a specific scripts tag with a `super()`